app/uk/gov/hmrc/gform/models/FieldValueMaker.scala

```diff
@@ -87,13 +87,13 @@ class FieldValueMaker(json: JsValue) {
 
   private lazy val optMES: Opt[MES] = (submitMode, mandatory) match {
     //format: OFF
-    case (_, _) if isThisAnInfoField => MES(mandatory = true, editable = false, submissible = false).asRight
-    case (Some(IsStandard()) | None, Some(IsTrueish()) | None)  => MES(mandatory = true, editable = true, submissible = true).asRight
-    case (Some(IsReadOnly()),        Some(IsTrueish()) | None)  => MES(mandatory = true, editable = false, submissible = true).asRight
-    case (Some(IsInfo()),            Some(IsTrueish()) | None)  => MES(mandatory = true, editable = false, submissible = false).asRight
-    case (Some(IsStandard()) | None, Some(IsFalseish()))        => MES(mandatory = false, editable = true, submissible = true).asRight
-    case (Some(IsInfo()),            Some(IsFalseish()))        => MES(mandatory = false, editable = false, submissible = false).asRight
-    case otherwise                                              => InvalidState(s"Expected 'standard', 'readonly' or 'info' string or nothing for submitMode and expected 'true' or 'false' string or nothing for mandatory field value, got: $otherwise").asLeft
+    case IsThisAnInfoField() => MES(mandatory = true, editable = false, submissible = false).asRight
```

Comment:
I've spent quite a bit time to make it formatted a nice way. Could you revert to old form?